cinder-client/src/main/java/com/woorea/openstack/cinder/SchedulerStatsExtension.java

```diff
@@ -0,0 +1,32 @@
+package com.woorea.openstack.cinder;
+
+import com.woorea.openstack.base.client.HttpMethod;
+import com.woorea.openstack.base.client.OpenStackClient;
+import com.woorea.openstack.base.client.OpenStackRequest;
+import com.woorea.openstack.cinder.model.Pools;
+
+/**
+ * Cinder Scheduler Stats Management
+ *
+ * @author Ahmed Saba
+ */
+public class SchedulerStatsExtension {
+
+    private final OpenStackClient CLIENT;
+
+    public SchedulerStatsExtension(OpenStackClient client) {
+        this.CLIENT = client;
+    }
+
+    public List list(boolean detail) {
+        return new List(detail);
+    }
+
+    public class List extends OpenStackRequest<Pools> {
+
+        public List(boolean detail) {
+            super(CLIENT, HttpMethod.GET, (new StringBuilder("/scheduler-stats/get_pools")).append(detail ? "?detail=True":""), null, Pools.class);
```

Comment:
Isn't it sufficient to just add 'detail'?
I.e. super(CLIENT, HttpMethod.GET, detail ? "/scheduler-stats/get_pools/detail" : "/scheduler-stats/get_pools", null, Pools.class);